apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java

```diff
@@ -79,6 +125,29 @@ String convertDcn(Parser.Data<String> data) {
     return "-" + DigestUtils.sha256Hex(claimId).substring(0, 22);
   }
 
+  @VisibleForTesting
+  void addDiagCodes(FissClaim.Builder claimBuilder, Parser.Data<String> data) {
+    for (int i = 1; i <= MAX_DIAG_CODES; ++i) {
+      final int INDEX = i;
+
+      // HHA and Hospice do not include procedure codes
+      data.get(Fiss.ICD_DGNS_CD + i)
+          .ifPresent(
+              value -> {
+                FissDiagnosisCode.Builder diagBuilder =
+                    FissDiagnosisCode.newBuilder()
+                        .setDiagCd2(data.get(Fiss.ICD_DGNS_CD + INDEX).orElse(""));
```

Comment:
Yep, the code a few lines up checks `data.get(Fiss.ICD_DGNS_CD + i).ifPresent(...)`, so it will only run this if there is actually a non-empty value, but for terseness I should be using `setDiagCd2(value);` here I think since I already pulled it, I'll fix that!